uh50_api/service.py

```diff
@@ -0,0 +1,22 @@
+"""
+Reads raw Heat Meter data and returns a HeatMeterResponse object
+"""
+from uh50_api.response import HeatMeterResponse
+
+
+class HeatMeterService:
+    """
+    Reads the heat meter and returns its value.
+    """
+
+    def __init__(self, reader) -> None:
+        self.reader = reader
+
+    def read(self) -> HeatMeterResponse:
+        raw_response = self.reader.read()
+        self.response = HeatMeterResponse(self.reader.model, raw_response)

```

Comment:
Waarom is response een field?

Reply:
Ik wil in de API de service een trap kunnen geven (read() ) en dan de reponse in datzelfde object terug kunnen zien. Als ik dat niet doe moet ik aan de voorkant een response-object bewaren en doorgeven. 
Het teruggeven van de response hoeft dan in feite niet, maar ik had ook niet het idee dat het overhead geeft. Hooguit verwarring (blijkt nu)

Reply:
Het is niet gebruikelijk om state bij te houden in een service. Het is sowieso een goed idee om gedrag en state zo veel mogelijk van elkaar gescheiden te houden. In dit geval misschien niet heel erg voor de hand liggend, maar stel je voor dat je meerdere clients/threads hebt die tegelijk die service gebruiken. Dan zou de ene call de response van de andere overschrijven. In een webapplicatie context is dit wel een erg waarschijnlijk usecase, maar in deze package waarschijnlijk niet. 
In je huidige implementatie wordt die self.response variabele alleen gebruikt binnen die methode, dus hoeft het niet persé een instance variable te zijn.

Reply:
response als field heb ik al verwijderd. Nu het ombouwen nog, want nu kan ik ook met fields(reponse_data) niet over de velden van de reponse wandelen